EsFormatter.py

```diff
@@ -119,10 +119,10 @@ def run(self, edit, save=False, ignoreSelection=False):
 
     def replaceFile(self, thread, save=False):
         '''Replace the entire file content with the formatted text.'''
+        sublime.status_message("File formatted")
```

Comment:
Because without this, there is no telling whether a formatting operation has completed or not.